actions/deploy/entrypoint.sh

```diff
@@ -19,16 +19,20 @@ if [ -z "$WAIT" ]; then
     export WAIT="true"
 fi
 
+if [ -z "$GITHUB_SHA" ]; then
+    export GIT_REF_SHA="$GITHUB_SHA"
+fi
+
 # Inject "image" as a template variable to a new copy of the vars file.
 # If the file doesn't exist, it is created. The original file is left untouched.
 if [ ! -z "$IMAGE" ]; then
     export VARS_ORIGINAL="$VARS"
     export VARS=`mktemp`
+    cat "$VARS_ORIGINAL" > "$VARS"
```

Comment:
What is the purpose?

Reply:
Reduce cyclomatic complexity https://en.wikipedia.org/wiki/Cyclomatic_complexity#Implications_for_software_testing
As mentioned in the commit message =)